docs/Using-Docker.md

```diff
@@ -95,7 +95,6 @@ docker run -it --name <container-name> \
            -p 5005:5005 \
```

Comment:
Despite this doc being deprecated, I've updated it in case users still find it and want to try it out.  This should also help us transition to a new Dockerfile in the future.